product_docs/docs/biganimal/release/getting_started/creating_a_cluster/index.mdx

```diff
@@ -85,8 +85,14 @@ Prior to creating your cluster, make sure you have enough resources. Without eno
        !!!tip
        To maximize your disk size for AWS, select Rb5 as your instance, and then io2 Block Express as your storage to get a maximum disk size of 64 TB and 256,000 IOPS.
   
-1.  In the **Storage** section, select your volume type from the **Volume Type** list. 
-       - For Azure, in **Volume Type** select either **Premium SSD** or **Ultra Disk**. Compared to Premium SSD volumes, ultra disks offer lower latency, high-performance options, and direct control over your disk's input/output operations per second (IOPS). For BigAnimal, we recommend using ultra disks for workloads that require the most demanding performance. See [Using Azure ultra disks](https://docs.microsoft.com/en-us/azure/virtual-machines/disks-enable-ultra-ssd?tabs=azure-portal) for more information. 
+1.  In the **Storage** section, select your volume type from the **Volume Type** list. For Azure, you can optionally choose to use a separate volume for write-ahead logs (WAL). 
+       
+       - For Azure, 
```

Comment:
you can provision a WAL volume for both Azure and AWS clusters. I see now that the designs only include Azure examples; this is an error. Both CSPs should be supported here.